scrapinghub/client.py

```diff
@@ -1018,3 +1061,10 @@ def get(self, key, *args, **kwargs):
         if key is None:
             raise ValueError("key cannot be None")
         return self._origin.get(key, *args, **kwargs)
+
+    def set(self, *args, **kwargs):
```

Comment:
hm, this method is still mentioned in the `proxy_methods(self._origin, self, [...])` call https://github.com/scrapinghub/python-scrapinghub/pull/48/files#diff-797cd9f74e9304996bef041260c04262R1044